analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -867,6 +867,11 @@ def main(args):
     LOG.debug("args: %s", str(args))
     LOG.debug("Output will be stored to: '%s'", args.output_path)
 
+    clang_binary = \
+        context.analyzer_binaries.get(
+            clangsa.analyzer.ClangSA.ANALYZER_NAME)
```

Comment:
Which `clang` specifically? What about `clang-tidy`? Do we assume that they are always the same version in the same directory?

Reply:
this version information is from the clang version which will be used for analysis (based on the config file values, package relative or from the PATH ...)
if clang-tidy has some problem during analysis because of the version mismatch (compiler clang, analyzer clang-tidy) probably there will be a problem with the analyzer clang too, this change just extends the collected information from the compiler clang

Reply:
Probably, but it's not a requirement to run Clang SA-based analysis, or even have the `clang` binary installed.

Reply:
yes, but if clang is not installed it will not be logged and so the implicit include paths will not be collected for it, they will be collected for the used compiler and used if clang-tidy is executed